### DISCUSSION TITLE
Nuxt app transpile dependency package.

### DISCUSSION THREAD

**Discussion by lukasborawski** [pnpm/pnpm#4975] [Q&A]
Hello. 

So I have the **Nuxt** app along with the **Turborepo** and **PMPN**, and the well-known issue with `Unexpected token '!'` from the **vue-server-renderer.** Trying to use transpile option for this package within the config however, this package is not a part of **node_modules** of the Nuxt app and it can't be recognized (I guess bc it's in the **pnpm** dependencies). Does anybody had the same issue, or maybe some suggestions now? Thanks for the help in advance.

**Comment by pencilcheck** [comment]
having the same issue
